openff/evaluator/protocols/forcefield.py

```diff
@@ -1052,3 +1063,54 @@ def _execute(self, directory, available_resources):
             )
 
         super(BuildTLeapSystem, self)._execute(directory, available_resources)
+
+
+@workflow_protocol()
+class BuildFoyerSystem(TemplateBuildSystem):
+    """Parameterize a set of molecules with a Foyer force field source"""
+
+    def _parameterize_molecule(self, molecule, force_field_source, cutoff):
+        """Parameterize the specified molecule.
+
+        Parameters
+        ----------
+        molecule: openff.toolkit.topology.Molecule
+            The molecule to parameterize.
+        force_field_source: FoyerForceFieldSource
+            The foyer source which describes which parameters to apply.
+
+        Returns
+        -------
+        openmm.System
+            The parameterized system.
+        """
+        import mdtraj as md
+        from foyer import Forcefield
```

Comment:
I go back and forth on whether an alias like this does more harm (longer class names) than good (making it clear which package a similarly-named class if from). What do you think?
```suggestion
        from foyer import Forcefield as FoyerForcefield
```